scalableminds/webknossos
# cross origin api sends acc message when done

This PR changes the cross origin api. It now sends an acknowledgment message after finishing it's work.

Edit Daniel: I also added that the api call is indeed awaited before acknowledging it. Additionally, error messages are sent if errors occur.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- send request to cross origin api
- when ready, an `ack` message should be send

### Issues:
- fixes #5552 

------
